src/gsi_ncdiag/gsi_ncdiag.py

```diff
@@ -1803,12 +1806,8 @@ def toIODAobs(self, OutDir, clobber=True):
         varDict[vname]['errKey'] = vname, iconv.OerrName()
         varDict[vname]['qcKey'] = vname, iconv.OqcName()
         VarDims[vname] = ['Location']
-        if (self.sensor in oz_lev_sensors):
-            varAttrs[varDict[vname]['valKey']]['units'] = 'mol mol-1'
-            varAttrs[varDict[vname]['errKey']]['units'] = 'mol mol-1'
-        else:
-            varAttrs[varDict[vname]['valKey']]['units'] = 'DU'
-            varAttrs[varDict[vname]['errKey']]['units'] = 'DU'
+        varAttrs[varDict[vname]['valKey']]['units'] = units_values[vname]
```

Comment:
@emilyhcliu @karpob I switched to just using units_values to set the units so now it just depends on the name. Then I added the complete list of ozone variables to units_values. I don't think this is a change from what was happening before but perhaps the code is a bit clearer?